src/utils/resolveUrl.js

```diff
@@ -8,7 +8,7 @@ const resolveUrl = (baseUrl, relativeUrl) => {
   }
 
   // if the base URL is relative then combine with the current location
-  if (!(/\/\//i).test(baseUrl)) {
+  if (!(/\/\//i).test(baseUrl) && window.location && window.location.href) {
```

Comment:
node does not have `window.location or window.location.href`

Reply:
Would we need to check `window &&` as well?